### PR TITLE
Bugfix: No longer crash SearchResultsComponent, if params.q is undefined

### PR DESCRIPTION
**Pull request by apepper**
The error can be observed, if one calls "npm run prerender".

Introduced with https://github.com/Scrivito/scrivito_example_app_js/pull/352.

### CODE REVIEW

File: src/Objs/SearchResults/SearchResultsComponent.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ class SearchResultsComponent extends React.Component {
   }
 
   query() {
-    return this.props.params.q.trim();
+    return this.props.params.q ? this.props.params.q.trim() : "";
```
Comment (dcsaszar) [inline src/Objs/SearchResults/SearchResultsComponent.js:90]: * [ ] FYI/Q: Can we shorten this to `this.props.params.q?.trim();`?

**Review by dcsaszar — APPROVED** [review]
LGTM

File: src/Objs/SearchResults/SearchResultsComponent.js (hunk 1 above)
Comment (apepper) [reply src/Objs/SearchResults/SearchResultsComponent.js:90]: If we simple do `this.props.params.q?.trim();` `query()` could return `undefined`. Components of inner props actually expect string values, e.g. `SearchResultItem`: https://github.com/Scrivito/scrivito_example_app_js/blob/a4e671110e0958e3071409f795775f3bdd2e6340/src/Objs/SearchResults/SearchResultItem.js#L50-L51

Comment (dcsaszar) [reply src/Objs/SearchResults/SearchResultsComponent.js:90]: FYI If there is no query this path is not taken.
